code/modules/mob/mob.dm

```diff
@@ -1334,7 +1334,7 @@ mob/proc/yank_out_object()
 		to_chat(host, "<span class='info'>You are now a mouse. Try to avoid interaction with players, and do not give hints away that you are more than a simple rodent.</span>")
 
 /mob/proc/assess_threat() //For sec bot threat assessment
-	return
+	return 5
```

Comment:
No, before it was simply set to set the threat value to 5 unless the mob had a specific threat value.